docs/organization/integrations/feature-flag/launchdarkly/index.mdx

```diff
@@ -23,3 +25,27 @@ The LaunchDarkly integration is only available to organizations with a Business
 1. Navigate to **Settings > Integrations > LaunchDarkly**
 
 2. Follow the full [LaunchDarkly installation instructions](https://docs.launchdarkly.com/integrations/sentry).
+
+## Evaluation Tracking
+
+Sentry can track flag evaluations as they happen within your application.  Flag evaluations will appear in the "Feature Flag" section of Issue Details page as a table, with "suspect" flag predictions highlighted in yellow. Learn more about how to interact with feature flag insights within the Sentry UI by reading the [Issue Details page documentation](/product/issues/issue-details/#feature-flags).
+
+### Set Up Evaluation Tracking
+
+To set up evaluation tracking visit one of our supported languages documentation page:
+* [JavaScript](/platforms/javascript/feature-flags/)
+* [Python](/platforms/python/feature-flags/)
+
+## Change Tracking
+
+Sentry can track changes to feature flag definitions and report suspicious feature flag edits.
+
+### Install and Configure
+
+To enable the change tracking integration click the "Set Up Integration" button on the feature flag table contained within the issue details page. This will trigger a fly out which will guide you through the two step process.
+
+![Integration UI](./img/ff-integration-ui.png)
+
+Under the "Integrate Feature Flag Service" heading a Sentry webhook URL has been provided. Additionally, a link to LaunchDarkly's webhook registration interface has been provided. Paste the Sentry webhook URL into the input box labeled "URL" within the LaunchDarkly webhook registration UI.
```

Comment:
I'm noticing now that the copywriting in the modal is very generic, but the link is for launchdarkly.com. We might want to change that link from "feature flag service" to clearly say "launch darkly", but that's code in another repo

Reply:
There is an image in another PR that didn't make it into here: https://github.com/getsentry/sentry-docs/pull/11749/files#diff-f09a4ba0a7ec1f45114b4bed5b636d218f53f9b9a5187b861f20d9a97140b970

The "Generic Trigger" screenshot from LD's website. that could be an asset to bring over and improve the setup steps here. For me, having the structure set (which we do!) is enough for now, we can iterate on the specific steps within this page later.

Reply:
Added a URL to the webhook page.  Took a note about the copywriting will alert michelle.